### PR TITLE
Update notices

### PR DESCRIPTION
**Pull request by fabianlindfors**
Ran `go mod tidy` to update dependencies and updated notices from https://docs.google.com/spreadsheets/d/1Aoh-IfylPMe5QpZnsRGS1dmQnBIioj6BH-aJLxhGDo4

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -5,7 +5,9 @@ github.com/Azure/go-autorest v11.1.2+incompatible/go.mod h1:r+4oMnoxhatjLLJ6zxSW
 github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46/go.mod h1:3wb06e3pkSAbeQ52E9H9iFoQsEEwGN64994WTCIhntQ=
 github.com/PuerkitoBio/purell v1.0.0/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbtSwDGJws/X0=
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
+github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 h1:DDGfHa7BWjL4YnC6+E63dPcxHo2sUxDIu8g3QgEJdRY=
```
Comment (jamesbroadhead) [inline go.sum]: license?

File: go.sum [hunk 2]
```diff
@@ -5,7 +5,9 @@ github.com/Azure/go-autorest v11.1.2+incompatible/go.mod h1:r+4oMnoxhatjLLJ6zxSW
...
 github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869/go.mod h1:Ekp36dRnpXw/yCqJaO+ZrUyxD+3VXMFFr56k5XYrpB4=
+github.com/bmizerany/pat v0.0.0-20170815010413-6226ea591a40 h1:y4B3+GPxKlrigF1ha5FFErxK+sr6sWxQovRMzwMhejo=
```
Comment (jamesbroadhead) [inline go.sum]: license?

File: go.sum [hunk 3]
```diff
@@ -19,6 +21,7 @@ github.com/elazarl/goproxy v0.0.0-20170405201442-c4fc26588b6e/go.mod h1:/Zj4wYkg
 github.com/emicklei/go-restful v0.0.0-20170410110728-ff4f55a20633/go.mod h1:otzb+WCGbkyDHkqmQmT5YD2WR4BBwUdeQoFo8l/7tVs=
 github.com/evanphx/json-patch v0.0.0-20190203023257-5858425f7550/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
 github.com/evanphx/json-patch v4.2.0+incompatible/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
+github.com/fsnotify/fsnotify v1.4.7 h1:IXs+QLmnXW2CcXuY+8Mzv/fWEsPGWxqefPtCP5CnV9I=
```
Comment (jamesbroadhead) [inline go.sum]: license?

File: go.sum (hunk 1 above)
Comment (fabianlindfors) [reply go.sum]: This one is stated as MIT licensed but there is no license file in the project. How do we handle that?

File: go.sum (hunk 2 above)
Comment (fabianlindfors) [reply go.sum]: Added!

File: go.sum (hunk 3 above)
Comment (fabianlindfors) [reply go.sum]: Added!

File: go.sum (hunk 1 above)
Comment (fabianlindfors) [reply go.sum]: Ping @jamesbroadhead

Comment (jamesbroadhead) [reply go.sum]: The project says that it's abandoned -- honestly, the best approach is probably not to use it. 

Failing that, include a copy of the MIT license
https://opensource.org/licenses/MIT

**Review by jamesbroadhead — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (BenElgar) [reply go.sum]: It's a transitive dependency I'm afraid. Are we keeping track of the licenses of those?

**Comment by jamesbroadhead** [comment]
Yes

**Pull request closed without merging**